nbs/models.nbeats.ipynb

```diff
@@ -40,8 +40,7 @@
     "This method proved state-of-the-art performance on the M3, M4, and Tourism Competition datasets, improving accuracy by 3% over the `ESRNN` M4 competition winner.\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        """ NBEATS
</pre><p>Nbdev best practices suggests keeping docstring short and elaborate in different cels: https://nbdev.fast.ai/tutorials/best_practices.html#keep-docstrings-short-elaborate-in-separate-cells</p><p>What is our take on that? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/297/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
We are not using documentation for the classes and methods as created by `nbdev`, because it leaves the code without docstrings.

We are waiting for `nbdev` to allow for the automatic generation of docstrings, which might simplify things in the future.